students/woogyeong/westagram/users/views.py

```diff
@@ -11,38 +12,41 @@
 
 class SignUpView(View):
     def post(self, request):
-        data     = json.loads(request.body)
-        email    = data['email']
-        password = data['password']
+        data = json.loads(request.body)
         
         try:
+            email     = data['email']
+            password  = data['password']
+            hashed_pw = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
변수명은 줄이지 말고 최대한 명확하게 표현해 주세요.
```suggestion
            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Reply:
지금 경래님 수업 들으면서 수정해야겠다 싶었는데 짚어주셨네요 감사합니다!
수정해서 다시 푸쉬했습니다.